modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * @file
+ * OE Whitelabel theme extra project.
+ */
+
+declare(strict_types =  1);
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\media\MediaInterface;
+use Drupal\media\Plugin\media\Source\Image;
+use Drupal\media\Plugin\media\Source\OEmbed;
+use Drupal\oe_bootstrap_theme\ValueObject\ImageValueObject;
+use Drupal\Component\Utility\Html;
+
+/**
+ * Implements hook_preprocess().
+ */
+function oe_whitelabel_extra_project_preprocess_pattern_description_list(array &$variables): void {
```

Comment:
I also dont like this, better to try suggestions for pattern templates and create a template for each group needed, there we can change the attribute.
see https://github.com/openeuropa/oe_theme/commit/2075b9d4748a4f1089df6eb278dfb9d28848949f

Reply:
we discussed we will keep this solution for now